### ISSUE TITLE
compromised lock throws error when release is called

### ISSUE THREAD

**Issue by pimlie** [moxystudio/node-proper-lockfile#81]
Is it intended behaviour that calling `release` for a lock which has been compromised throws an error? It feels counter-intuitive to me. Maybe we could add a `strict` mode, which is default true but when its false a compromised lock doesnt throw an error?

**Comment by satazor** [comment]
I agree. Can you change to now throw? This will have to be a major though because it’s a breaking change.

**Comment by linonetwo** [comment]
It always throws "Lock is already released", will it be solved?
